src/main/java/org/folio/services/item/ItemService.java

```diff
@@ -94,7 +94,9 @@ public ItemService(Context vertxContext, Map<String, String> okapiHeaders) {
     @SuppressWarnings("all")
     final List<Future> setHridFutures = items.stream()
       .map(item -> {
-        item.getStatus().setDate(itemStatusDate);
+        if (item.getStatus().getDate() == null) {
+          item.getStatus().setDate(itemStatusDate);
+        }
```

Comment:
Why did this need to change during this work?

Reply:
Otherwise all items will have the status date overwritten, I don't think this is something expected. 

I also think there was a bug in this piece if code, because the butch sync API can also handle batch updates, but the date should be populated only on create, AFAIK.

Reply:
I agree that this is likely a bug, thank you for discovering and fixing it